File: tools/bench/answers/python_sonnet/s1/06_stats.py
lst = [4, 8, 15, 16, 23]
mn = lst[0]
mx = lst[0]
for x in lst:
    if x < mn:
        mn = x
    if x > mx:
        mx = x
mean = sum(lst) / len(lst)
print(mn)
print(mx)
print(mean)

File: tools/bench/answers/go_sonnet/s1/06_stats.go
package main

import "fmt"

func main() {
	nums := []float64{4, 8, 15, 16, 23}
	min, max := nums[0], nums[0]
	sum := 0.0
	for _, n := range nums {
		if n < min {
			min = n
		}
		if n > max {
			max = n
		}
		sum += n
	}
	mean := sum / float64(len(nums))
	fmt.Println(min)
	fmt.Println(max)
	fmt.Println(mean)
}
